packages/strapi-connector-mongoose/lib/queries.js

```diff
@@ -419,10 +419,7 @@ module.exports = ({ model, strapi }) => {
   function count(params) {
     const filters = convertRestQueryParams(params);
 
-    return buildQuery({
-      model,
-      filters: { where: filters.where },
-    }).count();
+    return buildQuery({ model, filters: omit(['sort', 'limit', 'start'], filters) }).count();
```

Comment:
why not do it before the convert so we don't process the params at all ?

Reply:
:+1: